packages/adobe-sign/__tests__/repos/agreements.repo.spec.ts

```diff
@@ -0,0 +1,24 @@
+import "reflect-metadata";
+import { DynamoProvider, Mock } from "asu-core";
+import { AgreementsRepo } from "../../src/repos/agreements.repo";
+import { Template } from "../../src/models/asu/template";
+import { DynamoDB, Service, Request } from "aws-sdk";
+import { GetItemOutput } from "aws-sdk/lib/dynamodb/types";
+import { AWSError } from "aws-sdk/lib/error";
+
+describe('AgreeementsRepo', () => {
```

Comment:
```suggestion
describe('AgreementsRepo', () => {
```
Agreeeeeeements, lol